xml/admin_nfsganesha.xml

```diff
@@ -51,9 +47,29 @@
   <title>Installation</title>
 
   <para>
-   For installation instructions, see <!-- DEAD_LINK -->.
+   For installation instructions, see <xref linkend ="deploy-cephadm-day2-service-nfs"/>.
   </para>
  </sect1>
+
+ <sect1 xml:id="ceph-nfsganesha-namespace">
+   <title>Namespace Conventions</title>
```

Comment:
```suggestion
   <title>&ogw Namespace Conventions</title>
```

we should probably be careful to distinguish :
* RADOS pool namespace
* RGW namespace